src/components/UserProfile/UserProfileHeaderContent.tsx

```diff
@@ -11,45 +11,44 @@ import {
   Typography,
 } from "@mui/material";
 import { makeStyles, styled } from "@mui/styles";
-import { Theme } from "@mui/system";
 import { SocialInfo } from "../../interfaces/UserInterfaces";
 import { userRoleConvertor } from "../../utils/userRoles";
 
 const CustomTypographyPrimary = styled(Typography)({
-  fontSize: "25px",
-  fontWeight: "500",
-});
-
-const CustomTypographySecondary = styled(Typography)({
-  color: "#818386",
-  fontSize: "25px",
-  fontWeight: "500",
-});
-
-const useStyles = makeStyles((theme: Theme) => ({
-  socialMediaIcon: {
-    "&:hover": {
-      color: "blue",
+    fontSize: "25px",
+    fontWeight: "500",
+  }),
+  CustomTypographySecondary = styled(Typography)({
+    color: "#818386",
+    fontSize: "25px",
+    fontWeight: "500",
+  }),
+  useStyles = makeStyles(() => ({
+    socialMediaIcon: {
+      "&:hover": {
+        color: "blue",
+      },
     },
-  },
-}));
+  }));
 
 interface UserProfileDisplayProps {
   email: string;
   displayName: string;
   profileImageUrl: string;
-  socialInfo: string;
+  socialInfo: SocialInfo | string;
```

Comment:
What is `| string` used for, shouldn't every request be SocialInfo?

Reply:
This component is used in other 2 components. One of them fetches the user from a service with the social info already deserialized, the other gets it encoded. In order to support both usages, I made this change. I know that ideally we should receive the data in a consistent manner from the API and not as we do now, with JSON strings embedded in JSONs, but fixing these issues right now would get us out of the scope of this feature.